src/utils/auth.test.ts

```diff
@@ -1,22 +1,22 @@
 import { AxiosPromise, AxiosResponse } from 'axios';
 
-const { remote } = require('electron');
-const BrowserWindow = remote.BrowserWindow;
```

Comment:
We shouldn't call the constructor here for idempotency reasons. it was subpar as it was though, so we should probably be doing:


```
describe('utils/auth.tsx', () => {
    let w: BrowserWindow;
    
    beforeEach(() => {
      w = new BrowserWindow();
    });
    
    afterEach(() => {
      if (w) {
        w.destroy();
        w = null;
      }
    });
})

Reply:
well BrowserWindow is a mock anyway, and it always return the same instance
(see [src/__mocks__/electron.js](https://github.com/manosim/gitify/pull/538/files#diff-ba7ff5d703b63d792d86ea2b35d94465df0b1364cf046a1750292bded7ac59ac))

thats how it behaved before my changes, it would always return the same object (which wasnt even a class)
I didn't change that behavior

I guess we could remove the single instance check from the BrowserWindow mock, but I'm not sure I see the point since there isn't anything unique there